src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
-body {
+@import "./utils/mixins.scss";
+@import "./utils/variables.scss";
+@import "./block/header.scss";
+@import "./block/section.scss";
+@import "./block/starts.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document

Reply:
still not resolved